app/pods/cases/index/template.hbs

```diff
@@ -0,0 +1,222 @@
+{{page-title (t "cases")}}
+<Cases::CasesHeader 
+  @onSave={{this.close}} 
+  @showArchived={{this.showArchived}} 
+  @onToggleShowArchived={{this.toggleShowArchived}} 
+/>
+<div class="auk-scroll-wrapper">
+  <div class="auk-scroll-wrapper__body">
+    <div class="auk-u-m-4">
+      {{#if this.model.length}}
+        <table class="auk-table auk-table--styled js-data-table">
+          <thead>
+            <tr>
+              {{#each this.tableColumns as |column|}}
+                {{#if (get this.tableColumnDisplayOptions column.keyName)}}
+                  {{#if column.sortKey}}
+                    <Utils::ThSortable
+                      @onChange={{this.sortTable}}
+                      @currentSorting={{this.sort}}
+                      @field={{column.sortKey}}
+                      @label={{t column.translationKey}}
+                    />
+                  {{else}}
+                    <th class="auk-u-text-nowrap" >
+                      {{t column.translationKey}}
+                    </th>
+                  {{/if}}
+                {{/if}}
+              {{/each}}
+              <th>&nbsp;</th>
+            </tr>
+          </thead>
+          <tbody>
+            {{#if this.isLoadingModel}}
+              <tr>
+                <td colspan={{add this.tableColumns.length 1}}>{{t "loading"}}</td>
+              </tr>
+            {{else}}
+              {{#each this.model as |case|}}
+                <tr class="auk-table--interaction-cursor" {{on "click" (fn this.onClickTableRow case)}}>
+                  {{#if this.tableColumnDisplayOptions.caseName}}
+                    <td>
+                      {{!-- Fallback to case.title is required for legay data in production --}}
+                      {{if case.shortTitle case.shortTitle case.title}}
+                    </td>
+                  {{/if}}
+                  {{!-- Add later --}}
+                  {{!-- {{#if this.tableColumnDisplayOptions.latestOnAgenda}}
+                    <td>
+                      {{moment-format case.created "DD-MM-YYYY HH:mm"}}
+                    </td>
+                  {{/if}} --}}
+                  {{#if this.tableColumnDisplayOptions.createdOn}}
+                    <td>
+                      {{moment-format case.created "DD-MM-YYYY HH:mm"}}
+                    </td>
+                  {{/if}}
+                  <td>
+                    <div class="auk-button-toolbar">
+                      {{#if this.currentSession.isEditor}}
+                        <button type="button"
+                          class="auk-button auk-button--borderless auk-button--icon">
+                          <span class="auk-button__content">
+                            <div class="auk-icon auk-icon--default auk-icon--more">
+                              <i class="ki ki-more"></i>
+                            </div>  
+                          </span>
+                          <AttachPopover
```

Comment:
button naar auk component
Attachpopover mag naast de button staan en dat gaat ook werken (momenteel zit dat in die button)
```
<Auk::Button @skin="borderless" @icon="more" @layout="icon-only"/>
    <AttachPopover
            @renderInPlace={{true}}
            @class="ember-attacher-popper"
            @hideOn="clickout click"
            @showOn="click"
            @animation="shift"
            @placement="bottom-end"
    >
```